### ISSUE TITLE
Manga nook epubs can't be DeDRMed?

### ISSUE THREAD

**Issue by digitaldiatribe** [noDRM/DeDRM_tools#76]
Hi there, I installed the nook app from the Window store, logged in downloaded 2 books from my account, closed the app and went to work to pull my cchash and imported into the DeDRM plugin. After that, I imported one book (text only) to test and it went through no problem, the other book, a much larger manga volume, unfortunately did not. According to the debug log it says "is neither an Adobe Adept nor a Barnes & Noble encrypted ePub". What's the issue there? Generated a new debug log to show both import attempts.

[dedrm_debug.txt](https://github.com/noDRM/DeDRM_tools/files/8685175/dedrm_debug.txt)

**Comment by ElleKayEm** [comment]
Some books from the Windows nook app have a piece of their DRM info stored in a database instead of inside the epub itself.  See my comment here:  https://github.com/apprenticeharper/DeDRM_tools/issues/693#issuecomment-442184522

**Comment by digitaldiatribe** [comment]
Ah, I familiar with the image extraction workaround from my the manga I bought from Amazon in the past. Not totally ideal but I don't plan on buying many books from nook. Just filling holes of books Kobo doesn't sell. Thanks!

Since it sounds like a this is a Windows specific problem, would I be better off just doing the Android method to sidestep the whole issue?

**Comment by ElleKayEm** [comment]
Yes, it's possible that the manga downloaded with the Android app might not have this problem.  Let us know how it goes if you try that.

**Comment by jmdavis** [comment]
> Some books from the Windows nook app have a piece of their DRM info stored in a database instead of inside the epub itself. See my comment here: [apprenticeharper#693 (comment)](https://github.com/apprenticeharper/DeDRM_tools/issues/693#issuecomment-442184522)

This did not work for me. Even with the rights.xml file added, the files were all garbled after calibre processed them. A book that was not manga worked just fine, so the basic decryption works, but whatever is going on with the manga epubs, I haven't been able to get them to decrypt properly.

However, if I grab the files from an Android device rather than Windows, those manga epub files are able to be decrypted just fine in calibre. So, that workaround works, but it's obviously more of a pain than it would be if the manga epub files downloaded by the Windows app could be decrypted, and who knows if the Android app will eventually be changed to match. As it is, it used to be the case that they didn't even encrypt the files on Android (I guess that they assumed that you couldn't get at them, since the device has to be rooted to access them). So, I'd really like to see this problem resolved at some point, but I certainly don't have the expertise required to figure it out.

**Comment by ElleKayEm** [comment]
I wonder if it's some non-standard formatting which only the Windows Nook app knows how to display properly.  I know that when I did this with Nook comics, I ended up just pulling out the images.  See this comment at the old Alf blog:  https://apprenticealf.wordpress.com/2012/09/10/drm-removal-tools-for-ebooks/comment-page-38/#comment-62274

**Comment by jmdavis** [comment]
The images files are garbage after the decryption, as are the various XML files.

**Comment by ElleKayEm** [comment]
Ok, so not actually decrypting.  Are you able to decrypt other "regular" epubs from the Windows Nook app?  (If you have any.)  Maybe you have a different key for the Windows app.  There's a script here to get the Windows key:  https://github.com/noDRM/DeDRM_tools/discussions/9  See if that's the same as the one you obtained from the Android app.

**Comment by digitaldiatribe** [comment]
What I found for my decrypting was to make sure I got the correct block of text the xml file needs otherwise it won't work. It's fairly particular about it I'm afraid.
